packages/design-system/src/components/Button/Button.stories.jsx

```diff
@@ -94,6 +94,297 @@ VariationsOnDark.decorators = [
   ),
 ];
 
+export const AllAnchorButtons = () => {
+  return (
+    <>
+      <Button href="#!">Default anchor button</Button>
+
+      <div style={{ display: 'flex', gap: '1rem', flexDirection: 'column' }}>
+        <h4>Default and modified default</h4>
+        <div className="" style={{ display: 'flex', gap: '1rem', alignItems: 'center' }}>
+          <Button href="#!" size="big">
+            Anchor button
+          </Button>
+          <Button href="#!">Anchor button</Button>
+          <Button href="#!" size="small">
+            Anchor button
+          </Button>
+        </div>
+        <div className="" style={{ display: 'flex', gap: '1rem', alignItems: 'center' }}>
+          <Button size="big" disabled>
```

Comment:
For a link to be truly disabled, **a user cannot add an `href`**. We don't call this out in our docs and I don't know if we need to as it's part of the anchor spec. 

The reason this is important to note is that while we can visually style our link buttons to look like buttons, when a user wants a disabled anchor button it won't be a 1:1 match behaviorally to a disabled button button (i.e., AT can still access disabled anchor buttons).

Reply:
D'oh! I just realized when you exclude the `href` from something that was intended to be an anchor button, our component changes it into a button button - therefore the class we're using to style disabled anchor buttons is actually irrelevant. 

Do I remove `ds-c-button--disabled` from the codebase?